x/gov/types.go

```diff
@@ -0,0 +1,82 @@
+package gov
```

Comment:
the `types.go` folder is fine for now - but I think the governance module will be refactored similarly to how the staking module is about to be refactored where we will have a types directory, where each type and it's attribute functions can go in a separate file -additionally I think more functionality can be moved to be become attributes of the governance types - which would make the keepers even more straight forward.